internal/cli/performanceadvisor/slowquerylogs/list.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package slowquerylogs
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const listTemplate = `NAMESPACE{{range .SlowQuery}}
```

Comment:
[nit] we should include the log line

Reply:
this is how it looks like if I include LINE

```
./bin/mongocli om performanceAdvisor slowQueryLogs list --hostId 47bddeac69ab40bc578b0e5ae329f980 -P ops
NAMESPACE           LINE
admin.system.keys   {"t":{"$date":"2020-09-14T20:37:52.623+02:00"},"s":"I",  "c":"COMMAND",  "id":51803,   "ctx":"monitoring-keys-for-HMAC","msg":"Slow query","attr":{"type":"command","ns":"admin.system.keys","command":{"find":"system.keys","filter":{"purpose":"HMAC","expiresAt":{"$gt":{"$timestamp":{"t":1615646920,"i":0}}}},"sort":{"expiresAt":1},"readConcern":{},"$readPreference":{"mode":"nearest","tags":[]},"$db":"admin"},"planSummary":"COLLSCAN","keysExamined":0,"docsExamined":2,"hasSortStage":true,"cursorExhausted":true,"numYields":0,"nreturned":0,"queryHash":"6DC32749","planCacheKey":"6DC32749","reslen":231,"locks":{"ReplicationStateTransition":{"acquireCount":{"w":1},"acquireWaitCount":{"w":1},"timeAcquiringMicros":{"w":777822}},"Global":{"acquireCount":{"r":1}},"Database":{"acquireCount":{"r":1}},"Collection":{"acquireCount":{"r":1}},"Mutex":{"acquireCount":{"r":1}}},"readConcern":{"provenance":"clientSupplied"},"storage":{},"protocol":"op_msg","durationMillis":778}}
local.oplog.rs      {"t":{"$date":"2020-09-14T20:37:57.628+02:00"},"s":"I",  "c":"COMMAND",  "id":51803,   "ctx":"conn40","msg":"Slow query","attr":{"type":"command","ns":"local.oplog.rs","appName":"OplogFetcher","command":{"getMore":1958332123524905268,"collection":"oplog.rs","batchSize":13981010,"maxTimeMS":5000,"term":1,"lastKnownCommittedOpTime":{"ts":{"$timestamp":{"t":0,"i":0}},"t":-1},"$db":"local","$replData":1,"$oplogQueryData":1,"$readPreference":{"mode":"secondaryPreferred"},"$clusterTime":{"clusterTime":{"$timestamp":{"t":1600094920,"i":6}},"signature":{"hash":{"$binary":{"base64":"AAAAAAAAAAAAAAAAAAAAAAAAAAA=","subType":"0"}},"keyId":0}}},"originatingCommand":{"find":"oplog.rs","filter":{"ts":{"$gte":{"$timestamp":{"t":1600094909,"i":1}}}},"readConcern":{"level":"local","afterClusterTime":{"$timestamp":{"t":0,"i":1}}},"ntoreturn":13981010,"maxTimeMS":60000,"tailable":true,"awaitData":true,"oplogReplay":true,"term":1,"$db":"local","$replData":1,"$oplogQueryData":1,"$readPreference":{"mode":"secondaryPreferred"},"$clusterTime":{"clusterTime":{"$timestamp":{"t":1600094920,"i":6}},"signature":{"hash":{"$binary":{"base64":"AAAAAAAAAAAAAAAAAAAAAAAAAAA=","subType":"0"}},"keyId":0}}},"planSummary":"COLLSCAN","cursorid":1958332123524905268,"keysExamined":0,"docsExamined":0,"numYields":2,"nreturned":0,"reslen":678,"locks":{"ReplicationStateTransition":{"acquireCount":{"w":3},"acquireWaitCount":{"w":1},"timeAcquiringMicros":{"w":784275}},"Global":{"acquireCount":{"r":3}},"Database":{"acquireCount":{"r":3}},"Mutex":{"acquireCount":{"r":1}},"oplog":{"acquireCount":{"r":3}}},"readConcern":{"level":"local","afterClusterTime":{"$timestamp":{"t":0,"i":1}},"provenance":"clientSupplied"},"storage":{},"protocol":"op_msg","durationMillis":784}}
```

Reply:
Do you have any suggestions on how to make it better? @gssbzn

Reply:
It looks like is logging on a json format, I'm not sure if this is the default for atlas and ops manager but I don't mind the line being extra long, we haven't talk about truncating lines and I'm not sure we should